data/sampleSleepData.js

```diff
@@ -208,6 +208,36 @@ const sampleSleepData = [
     "date": "2019/06/21",
     "hoursSlept": 9.6,
     "sleepQuality": 4.1
+  },
+  {
+    "userID": 1,
+    "date": "2019/06/22",
+    "hoursSlept": 7,
+    "sleepQuality": 3
+  },
+  {
+    "userID": 2,
+    "date": "2019/06/22",
+    "hoursSlept": 4.8,
+    "sleepQuality": 3.3
+  },
+  {
+    "userID": 3,
+    "date": "2019/06/22",
+    "hoursSlept": 9.8,
+    "sleepQuality": 2.1
+  },
+  {
+    "userID": 4,
+    "date": "2019/06/22",
+    "hoursSlept": 7.7,
+    "sleepQuality": 1.5
+  },
+  {
+    "userID": 5,
+    "date": "2019/06/22",
+    "hoursSlept": 8.1,
```

Comment:
@Garrett-Iannuzzi Thank you for adding more data so we can test more thoroughly and ensure no false positives are coming through